spec/shardingSuitesConf.js

```diff
@@ -0,0 +1,37 @@
+// A suite of tests to run on max of two browsers instances at once, splitting test suites between
+// the two instances of chrome.
+var env = require('./environment.js');
+
+exports.config = {
+  seleniumAddress: env.seleniumAddress,
+
+  // Spec patterns are relative to this directory.
+  suites: {
+    okspec: 'suites/ok_spec.js',
+    okmany: ['suites/ok_spec.js', 'suites/ok_2_spec.js'],
```

Comment:
I'd like to see a test that actually made sure the sharding was happening.  E.g., in `okspec` set up some state which would cause `okmany` to fail, and have the second test in `okmany` only pass if the first one ran properly.